NetKAN/KashCorpSA.netkan

```diff
@@ -0,0 +1,20 @@
+{
+        "spec_version" : "v1.4",
+        "identifier" : "KashCorpSA",
+        "name" : "KashCorp Shared Assets (Virtual mod)",
+        "abstract" : "Shared assets for KashCorp mods, based off KAP",
+        "comment" : "KashCorp Shared Assets based off KAP zip file",
+        "license" : "BSD-3-clause",
+        "version" : "1.0v",
+        "$kref" : "#/ckan/kerbalstuff/96",
+        "depends" : [
+                { "name"        : "ModuleManager" }
+        ],
+        "install" : [
+                {
+                        "find" : "KashCorp",
+                        "install_to" : "GameData",
+                        "filter" : "KAP"
```

Comment:
Are the resources supposed to be installed by this mod or are they KAP specific?